lib/ansible/modules/cloud/amazon/aws_dynamodb_item.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, David C Martin @blastik
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: aws_dynamodb_item
+short_description: Creates, updates or deletes single items in AWS Dynamo DB tables.
+description:
+    - Creates, updates or deletes single items in AWS DynamoDB tables.
+    - More infomation can be found here U(https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#client).
+version_added: "2.9"
+author: "David C. Martin (@blastik)"
+options:
+    table:
+        description:
+            - Name of the DynamoDB table.
+        required: true
+        default: null
+        type: str
+    state:
+        description:
+          - If present, adds a new item or edits an existing item's attributes
```

Comment:
I'd add a note that without a conditional expression this operation is not idempotent.